hazelcast/src/test/java/com/hazelcast/map/EntryProcessorOffloadableTest.java

```diff
@@ -928,6 +938,110 @@ public String getExecutorName() {
         }
     }
 
+    /**
+     * <pre>
+     * Given: Heart beats are configured to come each few seconds (i.e. one quarter of hazelcast.operation.call.timeout.millis
+     *        - set in the {@code getConfig()} method)
+     * When: An offloaded EntryProcessor takes a long time to run.
+     * Then: Heart beats are still coming during the task is offloaded.
+     * </pre>
+     *
+     * @see #getConfig()
+     * @see #HEARTBEATS_INTERVAL_SEC
+     */
+    @Test
+    public void testHeartBeatsCommingWhenEntryPropcessorOffloaded() throws InterruptedException, IOException {
```

Comment:
Typo: "coming" vs. "comming"

I would also just do `throws Exception`, since it simplifies the test code (it doesn't matter which exception is thrown, the test will fail on any non-expected ones).